.devcontainer/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/java/.devcontainer/base.Dockerfile
 # [Choice] Java version (use -bullseye variants on local arm64/Apple Silicon): 8, 11, 16, 8-bullseye, 11-bullseye, 16-bullseye, 8-buster, 11-buster, 16-buster
```

Comment:
In the [Choice] list, we no longer provide `16` image, could we change `16` to `17`?
Other looks good to me.

Reply:
Good catch - should I just update the comment to https://github.com/microsoft/vscode-dev-containers/blob/main/containers/java/.devcontainer/base.Dockerfile? Or is there a reason we don't/shouldn't use the base Dockerfile on `main` rather than the `v0.195.0` branch?

Reply:
Yes, we could update the comment to https://github.com/microsoft/vscode-dev-containers/blob/main/containers/java/.devcontainer/base.Dockerfile. I guess v0.195 branch is latest when someone last edited this Dockerfile.

Reply:
Updated, thanks!